cloudigrade/util/tests/redhatcloud/test_sources.py

```diff
@@ -20,6 +25,11 @@ def setUp(self):
         self.account_number = str(_faker.pyint())
         self.authentication_id = _faker.user_name()
         self.application_id = _faker.pyint()
+        self.listener_server = "test_kafka_server"
+        self.listener_port = "19092"
+        self.sources_resource_type = "test_application"
+        self.sources_kafka_topic = "test_availability_status_topic"
+        self.sources_availability_event_type = "test_availability_status"
```

Comment:
Since this test suite already has a Faker instance being used just above this line, let me introduce you! 😀 

https://github.com/joke2k/faker#basic-usage

We try to avoid entering hard-coded values into our tests unless a specific situation calls for a persistent known value. This helps to ensure that we aren't _accidentally_ relying on the behavior of some static value in some unusual way. One possible use of Faker here might look like:

```
self.listener_server = _faker.hostname()
self.listener_port = str(_faker.pyint())
self.sources_resource_type = _faker.slug()
self.sources_kafka_topic = _faker.slug()
self.sources_availability_event_type = _faker.slug()
```